langchain4j-azure-open-ai/src/test/java/dev/langchain4j/model/azure/AzureOpenAiChatModelIT.java

```diff
@@ -322,6 +335,135 @@ void should_support_all_string_model_names(AzureOpenAiChatModelName modelName) {
         assertThat(response.content().text()).isNotBlank();
     }
 
+    @ParameterizedTest(name = "Deployment name {0} using {1}")
+    @CsvSource({
+        "gpt-4o,        gpt-4o"
+    })
+    void should_listen_request_and_response(String deploymentName, String gptVersion) {
```

Comment:
minor: `gptVersion` can be removed together with `.tokenizer(new AzureOpenAiTokenizer(gptVersion))`

Reply:
removed gptVersion and tokenizer call